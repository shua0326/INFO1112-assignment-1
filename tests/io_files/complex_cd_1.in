pwd
cd io_files
pwd
cd 
pwd
cd non_existant
pwd
ls
cd tests
pwd
