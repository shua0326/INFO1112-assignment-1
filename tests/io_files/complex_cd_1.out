>> /home/tests
>> >> /home/tests/io_files
>> >> /home
>> cd: no such file or directory: /home/non_existant
>> /home
>> mysh.py
parsing.py
__pycache__
tests
>> >> /home/tests
>> 
